.github/workflows/ci.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - run:
           # for git tests
           git config --global user.email "scalafmt@scalameta.org" && git config --global user.name "scalafmt"
-      - run: sbt plugin/scripted "++ 3.x" compile
+      - run: sbt "+ plugin/scripted"
```

Comment:
what did this do before, and doesn't anymore?

Reply:
before

- sbt 1.x: compile and test
- sbt 2.x: compile only

after

- sbt 1.x: compile and test
- sbt 2.x: compile and test

https://github.com/sbt/sbt/blob/9ead0ebfd517c00acf243cf610bcfdbf2c4997ef/main/src/main/scala/sbt/internal/CommandStrings.scala#L341-L352
https://github.com/sbt/sbt/blob/9ead0ebfd517c00acf243cf610bcfdbf2c4997ef/main/src/main/scala/sbt/internal/CommandStrings.scala#L333